core/db/migrate/20211203082008_add_settings_to_payment_methods.rb

```diff
@@ -0,0 +1,11 @@
+class AddSettingsToPaymentMethods < ActiveRecord::Migration[5.2]
```

Comment:
I checked previous migrations and they inherit from ActiveRecord::Migration[5.2] so I did the same here